tests/resources/helper_functions.py

```diff
@@ -217,6 +217,7 @@ def deploy_erc721_erc20(
     config: Config,
     erc721_publisher: Wallet,
     erc20_minter: Optional[Wallet] = None,
+    cap=Web3.toWei("0.5", "ether"),
```

Comment:
can you please specify the type of `cap` as well here? and also we need to provide `@enforce_type` to this function, that would be nice to have. Related to the `cap` value, would it work if we keep this as a constant in `web3_internal/constants.py`